frontend/app/components/auth/__email-login-form/auth__email-login-form.tsx

```diff
@@ -97,9 +97,8 @@ export class EmailLoginForm extends Component<Props, State> {
     try {
       await this.props.sendEmailVerification(this.state.usernameValue, this.state.addressValue);
       this.setState({ verificationSent: true });
-      setTimeout(() => {
-        this.tokenRef.current && this.tokenRef.current.focus();
-      }, 100);
+      await sleep(100);
```

Comment:
Hardcoded sleep instead of a timeout doesn't seem right.

Reply:
it's the same way as was used in this file and it is `setTimeout` wrapped in func